src/7bb60_len_41b0.c

```diff
@@ -110,42 +105,25 @@ void phys_init_integrator_for_current_state() {
 INCLUDE_ASM(void, "7bb60_len_41b0", phys_init_integrator_for_current_state);
 #endif // NON_MATCHING
 
-#ifdef NON_MATCHING
-// void gravity_use_fall_parms(void) {
-//     f32* floats = D_800F7B60;
-// do { } while (0);
-//     if (gPlayerStatus.flags & 0x40000) {
-//         gPlayerStatus.gravityIntegrator[0] = *floats++ / 12.0f;
-//         gPlayerStatus.gravityIntegrator[1] = *floats++ / 12.0f;
-//         gPlayerStatus.gravityIntegrator[2] = *floats++ / 12.0f;
-//         gPlayerStatus.gravityIntegrator[3] = *floats++ / 12.0f;
-//     } else {
-//         gPlayerStatus.gravityIntegrator[0] = *floats++;
-//         gPlayerStatus.gravityIntegrator[1] = *floats++;
-//         gPlayerStatus.gravityIntegrator[2] = *floats++;
-//         gPlayerStatus.gravityIntegrator[3] = *floats++;
-//     }
-// }
+// This function is wack. This weird stuff is needed to match
 void gravity_use_fall_parms(void) {
-    PlayerStatus* playerStatus;
     f32* floats = D_800F7B60;
+    PlayerStatus* playerStatus;
     do {} while (0);
     playerStatus = &gPlayerStatus;
+
     if (playerStatus->flags & 0x40000) {
-        playerStatus->gravityIntegrator[0] = (*(floats++)) / 12.0f;
-        playerStatus->gravityIntegrator[1] = (*(floats++)) / 12.0f;
-        playerStatus->gravityIntegrator[2] = (*(floats++)) / 12.0f;
-        playerStatus->gravityIntegrator[3] = (*(floats++)) / 12.0f;
+        playerStatus->gravityIntegrator[0] = *(floats++) / 12.0f;
```

Comment:
The parens around `(floats++)` aren't required here, I'd vote remove them like we do with `*args++` in script funcs.